include/clang/StaticAnalyzer/Core/AnalyzerOptions.def

```diff
@@ -355,6 +355,12 @@ ANALYZER_OPTION(StringRef, CTUIndexName, "ctu-index-name",
                 "the name of the file containing the CTU index of definitions.",
                 "externalDefMap.txt")
 
+ANALYZER_OPTION(StringRef, CTUCompilationDatabase, "ctu-compilation-database",
+                "The path to the compilation database used for on-demand "
+                "parsing of ASTs during CTU analysis. An empty valued disables "
```

Comment:
Why don't we give an error for the user if the value is empty? There are cli switches which do require a parameter, this is like that.

Reply:
Fixed. See CompilerInvocation.ccp.